### DISCUSSION TITLE
Jitsi Integration Documentation

### DISCUSSION THREAD

**Discussion by neferin12** [OpenSlides/OpenSlides#6665] [Q&A]
I hope I didn't miss anything, but I wasn't able to find any information on how I need to configure the server for the use of Jitsi for video conferencing. Could anyone give me a hint, where/how to do/enable that? 

(Btw, I love this project, thanks for this great piece of software)

**Comment by jsangmeister** [comment]
Unfortunately, we currently do not have such information. We will try to create and link it here.

**Comment by m-schieder** [comment]
Instructions can now be found in our [FAQ](
https://support.openslides.com/help/de-de/4-livestream-videokonferenz/206-konfiguration-der-jitsi-integration-im-selbsthosting).
